about.html

```diff
@@ -5,6 +5,43 @@
     <title>CSS Intro</title>
   </head>
   <body>
+    <style>
```

Comment:
Why did you decide to do this with a style tag in your html?

(For the remainder of core, we'll typically write css in separate files!)